### PR TITLE
Creating a K8s Injection Image

### PR DESCRIPTION
**Pull request by bengl**
Adds a GitHub action to create an init image, and test it.

In a future PR, we'll add the init image to our release process.

### CODE REVIEW

File: k8s/sample-app/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+console.log(process.env)
```
[the remark this answers is not in the document]
Comment (bengl) [reply k8s/sample-app/index.js]: Got rid of this in latest commit.

**Review by rochdev — APPROVED** [review]
The README could use a section that describes what the image is and what is does, but this can be done when the image is actually shipped.

**Comment by rochdev** [comment]
What makes this PR `semver-minor`?

**Comment by bengl** [comment]
> What makes this PR `semver-minor`?

Actually, come to thing of it, this is actually `semver-patch` until the release portion of this comes in the later PR. I'll relabel as such.

**Review by rochdev — APPROVED** [review]

File: k8s/pod.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - name: node-init
+      image: bengl/test-auto-inst:latest  # put the library image here
+      imagePullPolicy: Never
+      command: ["sh", "copy-lib.sh", "/autoinstrumentation"]
+      volumeMounts:
+        - name: autoinstrumentation
+          mountPath: /autoinstrumentation
+  containers:
+    - name: app
+      image: bengl/test-sample-app # put the app image here
```
Comment (uurien) [inline k8s/pod.yml:17]: There are multiple references pointing to `bengl/<something>` Is this OK? Or do we have to change it to something like `dd/test-sample-app`...

Comment (bengl) [reply k8s/pod.yml:17]: It's fine. Mentioned in the readme.

**Pull request merged**